src/sections/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Footer from "./components/Footer/SideBarFooter";
+import SelectedListItem from "./components/NavLinks/SideBarLinks";
+import ProfilePic from "./components/ProfilePic/SideBarProfile";
+import { styles } from "../../../../styles/styles";
```

Comment:
Please investigate how to define relative routes so that we can start importing files like from `styles` and `components/ProfilePic` also please create an index file on components level that will export all the contained components so that you can import it from `./components`. Also **All generic components** like SideBarProfile, or SideBarLink should be located on a folder named `components` at root level. they can be used or shared between different components so don't define it inside the components of a section, it should be a generic component you will be using in different files